packages/product/package.json

```diff
@@ -8,8 +8,6 @@
     "dist"
   ],
   "bin": {
-    "medusa-product-migrations-down": "dist/scripts/bin/run-migration-down.js",
-    "medusa-product-migrations-up": "dist/scripts/bin/run-migration-up.js",
```

Comment:
question: We won't have scripts any longer, only functions?

Reply:
yes, we use to have script cause it was not yet handled by the CLI, but now the CLI handle them plus I ve fixed the client url loading. So now when you run `medusa migrations run` it goes through those functions as they are grabbed from the module exports

Reply:
I ve only kept the seed script as this one is manual and you can pass a file path to it

Reply:
I ve run manually in debug to ensure that and it worked pretty well